src/components/panel/panel.js

```diff
@@ -1086,13 +1098,19 @@ MdPanelRef.prototype.removeClass = function(oldClass) {
  * Toggle a class on the panel. DO NOT use this to hide/show the panel.
  *
  * @param {string} toggleClass The class to toggle.
+ * @param {boolean} onElement Whether or not to toggle the class on the panel
```

Comment:
New params need to be added to the public documentation at the top of this file. (For add, remove, and toggle)